Project 10/index.js

```diff
@@ -0,0 +1,128 @@
+const input = document.getElementById('input');
+let result;
+let operation = '';
+let deleteIfEnter = false;
+let clearMemory = false;
+
+const operations = {
+	'' : (value) => sumResult(value),
+	'+': (value) => sumResult(value),
+	'-': (value) => sumResult(-value),
+	'*': (value) => multiplyResult(value),
+	'/': (value) => divideResult(value),
+	'=': (value) => showResult(value)
+}
+
+function onload() {
+	document.addEventListener("keydown", keyDownInput, false);
```

Comment:
```suggestion
	document.addEventListener("keydown", keyDownInput);
```
`useCapture` - опциональный параметр и по дефолту и так ставится в false